src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -14,15 +14,21 @@
 
 error_reporting(-1);
 
-$getEnvVar = function ($name, $default = false) {
+global $argv, $argc;
+$argv = isset($_SERVER['argv']) ? $_SERVER['argv'] : array();
+$argc = isset($_SERVER['argc']) ? $_SERVER['argc'] : 0;
+$getEnvVar = function ($name, $default = false) use ($argv) {
     if (false !== $value = getenv($name)) {
         return $value;
     }
 
     static $phpunitConfig = null;
     if (null === $phpunitConfig) {
+        $opt = min(array_search('-c', $opts = array_reverse($argv), true) ?: \INF, array_search('--configuration', $opts, true) ?: \INF);
```

Comment:
no need for the leading `\` (same below) for two reasons: this file is in the global namespace - and that's not the code style we use in Symfony ;)